spotbugs/src/main/java/edu/umd/cs/findbugs/util/CollectionAnalysis.java

```diff
@@ -0,0 +1,30 @@
+package edu.umd.cs.findbugs.util;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import edu.umd.cs.findbugs.ba.ClassMember;
+
+/**
+ * Utility class for analyzing collections.
+ */
+public final class CollectionAnalysis {
+
+    private CollectionAnalysis() {
+    }
```

Comment:
Can you please add a comment that it's empty because it's a utility class, so the Sonar analysis won't mark it as problematic?